```
[ERROR] Failures: 
[ERROR]   AllMetricsTest>Arquillian.run:138->testAllMetrics:83 » AbstractMethod Receiver...
[ERROR]   BulkheadMetricTest>Arquillian.run:138->bulkheadMetricAsyncTest:204->waitForQueuePopulation:240 » AbstractMethod
[ERROR]   BulkheadMetricTest>Arquillian.run:138->bulkheadMetricRejectionTest:133 » AbstractMethod
[ERROR]   BulkheadMetricTest>Arquillian.run:138->bulkheadMetricTest:99 » AbstractMethod ...
```

```
[ERROR] bulkheadMetricTest(org.eclipse.microprofile.fault.tolerance.tck.metrics.BulkheadMetricTest)  Time elapsed: 0.008 s  <<< FAILURE!
java.lang.AbstractMethodError: Receiver class io.smallrye.faulttolerance.metrics.MetricsCollectorFactory$MetricsCollectorImpl$$Lambda$466/0x0000000800592440 does not define or inherit an implementation of the resolved method abstract getValue()Ljava/lang/Object; of interface org.eclipse.microprofile.metrics.Gauge.
	at org.eclipse.microprofile.fault.tolerance.tck.metrics.util.MetricGetter.lambda$getGaugeValue$0(MetricGetter.java:343)
	at java.base/java.util.Optional.map(Optional.java:265)
	at org.eclipse.microprofile.fault.tolerance.tck.metrics.util.MetricGetter.getGaugeValue(MetricGetter.java:343)
	at org.eclipse.microprofile.fault.tolerance.tck.metrics.util.MetricGetter.getBulkheadConcurrentExecutions(MetricGetter.java:288)
	at org.eclipse.microprofile.fault.tolerance.tck.metrics.BulkheadMetricTest.bulkheadMetricTest(BulkheadMetricTest.java:99)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.arquillian.QuarkusProtocol$QuarkusMethodExecutor$1.invoke(QuarkusProtocol.java:80)
	at org.jboss.arquillian.container.test.impl.execution.LocalTestExecuter.execute(LocalTestExecuter.java:57)
	at jdk.internal.reflect.GeneratedMethodAccessor44.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
	at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
	at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:90)
	at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:133)
```

--- a/bom/runtime/pom.xml
+++ b/bom/runtime/pom.xml
@@ -33,10 +33,10 @@
         <microprofile-rest-client.version>1.3.4</microprofile-rest-client.version>
         <smallrye-config.version>1.3.9</smallrye-config.version>
         <smallrye-health.version>2.1.0</smallrye-health.version>
-        <smallrye-metrics.version>2.2.0</smallrye-metrics.version>
+        <smallrye-metrics.version>2.3.0</smallrye-metrics.version>
         <smallrye-open-api.version>1.1.19</smallrye-open-api.version>
         <smallrye-opentracing.version>1.3.2</smallrye-opentracing.version>
-        <smallrye-fault-tolerance.version>2.1.0</smallrye-fault-tolerance.version>
+        <smallrye-fault-tolerance.version>2.1.1</smallrye-fault-tolerance.version>
         <smallrye-jwt.version>2.0.9</smallrye-jwt.version>
         <smallrye-context-propagation.version>1.0.11</smallrye-context-propagation.version>
         <smallrye-reactive-streams-operators.version>1.0.10</smallrye-reactive-streams-operators.version>

--- a/build-parent/pom.xml
+++ b/build-parent/pom.xml
@@ -58,7 +58,7 @@
         <!-- MicroProfile TCK versions -->
         <microprofile-health-api.version>2.1</microprofile-health-api.version>
         <microprofile-config-api.version>1.3</microprofile-config-api.version>
-        <microprofile-metrics-api.version>2.1.0</microprofile-metrics-api.version>
+        <microprofile-metrics-api.version>2.2</microprofile-metrics-api.version>
         <microprofile-fault-tolerance-api.version>2.0.2</microprofile-fault-tolerance-api.version>
         <microprofile-reactive-messaging-api.version>1.0</microprofile-reactive-messaging-api.version>
         <microprofile-rest-client-api.version>1.3.4</microprofile-rest-client-api.version>

--- a/tcks/microprofile-fault-tolerance/tck-suite.xml
+++ b/tcks/microprofile-fault-tolerance/tck-suite.xml
@@ -32,24 +32,11 @@
                 </methods>
             </class>
 
-            <!-- https://github.com/eclipse/microprofile-fault-tolerance/issues/464 -->
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.BulkheadMetricTest">
-                <methods>
-                    <exclude name="bulkheadMetricAsyncTest"/>
-                    <exclude name="bulkheadMetricRejectionTest"/>
-                    <exclude name="bulkheadMetricTest"/>
-                </methods>
-            </class>
-            <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.AllMetricsTest">
-                <methods>
-                    <exclude name="testAllMetrics"/>
-                </methods>
-            </class>
             <class name="org.eclipse.microprofile.fault.tolerance.tck.metrics.CircuitBreakerMetricTest">
                 <methods>
                     <exclude name="testCircuitBreakerMetric"/>
                 </methods>
-            </class>
+            </class> 
 
         </classes>
     </test>

